### ISSUE TITLE
BSD support Net Free and PC

### ISSUE THREAD

**Issue by hlandau** [netbootxyz/netboot.xyz#78]
Would be nice.

**Comment by antonym** [comment]
Haven't had much success getting NetBSD working with iPXE yet.  This seemed close, but didn't work: http://www.syslinux.org/wiki/index.php/Mboot.c32

Any suggestions?

**Comment by Stealthii** [comment]
I had it netbooting once for next68k hardware, but it seems [Sal](http://boot.salstar.sk/)'s done work for getting iPXE working smoothly: http://boot.salstar.sk/bsd.ipxe

**Comment by thelamer** [comment]
Old issue I know, but cleaning up. 
I just loop tested 6 random images from that menu and none of them worked including netbsd, they all fail out and mount root stages of boot. 
I think this project has concentrated on linking ISOs up with memdisk but I am not sure the end to end testing that is performed.

**Comment by thelamer** [comment]
NetBSD is not possible, they use a custom chainload kernel that expects the kernel it loads to be in a specific file format on a TFTP server.

https://wiki.netbsd.org/tutorials/how_to_install__40__boot__41___netbsd_using_pxelinux/

FreeBSD is kind of the same story with needing NFS for official version, you can get to a splash memdisking the ISOs but you will get a mountroot failure as it looks for the squash in the CD device that does not exist. 

TrueOS ISO is huge, they have the same zipped kernel asset style as NetBSD, I think all of that base boot logic is identical. 

In summary we are just lucky OpenBSD works slightly using memdisk. IPXE booting will not happen for the rest.
